pkgs/development/libraries/libuninameslist/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, fetchFromGitHub
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libuninameslist";
+  version = "20190701";
+
+  src = fetchFromGitHub {
+    owner = "fontforge";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256:034c8clnskvqbwyiq7si4dad1kbngi3jmnrj064i39msqixmpdzb";
```

Comment:
not sure whether we allow SRI hashes in Nixpkgs yet. I think that will be fine after 20.03, but I may be wrong.

Reply:
There are already 12 occurences merged on master. But anyway, this is not important for me. I'll change and rebase to latest staging.

Reply:
You were right, it's fine.